### ISSUE TITLE
Output should be a device with entities not an entity with attributes

### ISSUE THREAD

**Issue by borpin** [dckiller51/bodymiscale#40]
The output from this should be a device with entities not an entity.

For instance, the attribute 'weight' should be an entity sensor with attributes for Unit_of_measurment.

Most of the current attributes, need to be entities with attributes specifying units as a minimum.

Really like it though!

**Comment by quizzical86** [comment]
Yes transforming the attributes into entities would enable having long term statistics about them.
https://data.home-assistant.io/docs/statistics/
You can only have long term statistics on the weight you give as an input to this custom component.

**Comment by Foxeronie** [comment]
+1 would be great to track these statistics
